lua/octo/pickers/fzf-lua/pickers/prs.lua

```diff
@@ -41,6 +41,8 @@ return function(opts)
         "api",
         "graphql",
         "--paginate",
```

Comment:
I have this causing an error saying it is not one of the available flags

Reply:
Good catch! Reverted this part.